it/uk/gov/hmrc/fhdds/testsupport/verifiers/DesVerifier.scala

```diff
@@ -9,7 +9,7 @@ import scala.collection.JavaConverters._
 case class DesVerifier()(implicit builder: VerifierBuilder) extends Matchers {
 
   def verifiesSubscriptions(): VerifierBuilder = {
-    val requests = findAll(postRequestedFor(urlMatching("/fhdds-stubs/application/.*"))).asScala
+    val requests = findAll(postRequestedFor(urlMatching("/fhdds-stubs/fulfilment-diligence/subscription/.*"))).asScala
```

Comment:
I don't think these urls are correct - they got updated with the latest version of the DES API. Please consult the API specification in our confluence space.